src/test/java/io/cdap/plugin/add/field/MultiFieldAdderConfigTest.java

```diff
@@ -32,6 +34,13 @@
   private static final MultiFieldAdderConfig VALID_CONFIG = new MultiFieldAdderConfig(
     "test_1:s_value_1,test_2:s_value_2,test_3:s_value_3"
   );
+  private static final String MACRO_VALUES = "test_1:${value1},test_2:value_2,test_3:${value_3}";
```

Comment:
please add unit tests for secure macro case, nested macros in the value, all key macro, none of the key macro, partial keys macro, nested macros in the key

Reply:
Tests are extended